### DISCUSSION TITLE
Can we use a subdomain, with a custom domain that already sends email to protonmail?

### DISCUSSION THREAD

**Discussion by starmaaan** [simple-login/app#1030] [Q&A] [answered]
Hi,

My maindomain.com (with Namecheap) already gets email delivered to Protonmail.

Is it possible that emails sent to subdomain.maindomain.com, are delivered to Simplelogin?

If so, does the normal domain setup work, while setting up a subdomain too?

Thanks in advance!

**Comment by nguyenkims** [comment, marked as answer]
@starmaaan yes you can add this subdomain on SimpleLogin. We have a dedicated guide for Namecheap on https://simplelogin.io/docs/custom-domain/registrars/namecheap/namecheap-subdomain/

**Reply by starmaaan** [reply]
Thank you @nguyenkims this helped me set up the subdomain correctly. My question was more around whether email service providers could both exist at the domain AND the subdomain level separately. Your replying to me suggested it could. Based on my test; it worked.

domain.com - goes to Protonmail
sub.domain.com - goes to Simplelogin
